docs/source/composer_model.rst

```diff
@@ -77,7 +77,7 @@ We also provide several common classes for various tasks, specifically:
    loss and accuracy metric.
 -  :class:`.TIMM` - creates classification models from the popular `TIMM`_
    library.
--  :class:`.ComposerTransformer` - base class for 🤗 `Transformers`_ models.
+-  :class:`.HuggingFaceModel` - converts a 🤗 `Transformers`_ model into a :class:`.ComposerModel`.
 
```

Comment:
nitpick: can we say "wraps" in a ComposerModel, or "creates a ComposerModel from a"? "Converts" sounds to me like the transformer model itself is being altered, maybe in some magical way I don't understand.

```suggestion
-  :class:`.HuggingFaceModel` - :class:`.ComposerModel` wrapper for a 🤗 `Transformers`_ model.

```